### PR TITLE
[Fix] User can delete any event (Front-end)

### LINKED ISSUE #832: [Security Bug] User can delete any event (Front-end)
**Describe the bug**
Normal user can delete events created by other user/admin

**To Reproduce**
1. Switch to Events Screen
2. Tap on Event (which is created by other user) 
3. Click on menu (3 dot) & Tap delete.

**Expected behavior**
User should not able to delete event created by others.

**Actual behavior**
User is able to delete others event

**Screenshots**

**Additional details**
Github Externship Appication No. : `20-05_Adi500_tfa_262`

Backed : Issue for API PalisadoesFoundation/talawa-api#261

### PR DESCRIPTION
**Pull request by AdityaBirangal**
**What kind of change does this PR introduce?**
fix #832

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
Not required

**Summary**
Front-end (Flutter) part fixed 
Back-end (API) will be get fixed in PalisadoesFoundation/talawa-api#261

**Does this PR introduce a breaking change?**
Nope

**Other information**
Github Externship Appication No. : `20-05_Adi500_tfa_262`

### CODE REVIEW

**Comment by TheHazeEffect** [comment]
I'm thinking Users shouldn't even see an option to delete if they are unable to,
What do you think? @palisadoes

**Comment by palisadoes** [comment]
Yes, that's true. The additional security of them not being able to do it inn the API is going to be added. I'll open a bug issue

**Comment by TheHazeEffect** [comment]
@AdityaBirangal could you go ahead and hide the button for those who are unable to delete?
Apart from that everything else is solid, I'll review and merge it right after

**Comment by AdityaBirangal** [comment]
It will be good as per users perspective if he/she didn't see an option to delete if he/she is not allowed.

I did required changes @TheHazeEffect 
Please review it & let me know if any updates are required...

**Comment by palisadoes** [comment]
@Sagar2366 @DeltaHarbinger Please take a look.

File: lib/views/pages/events/event_card_widget.dart [hunk 1]
```diff
@@ -105,24 +105,28 @@ Widget popUpMenue(
...
-      const PopupMenuItem<int>(
-          value: 4,
-          child: const ListTile(
-            leading: const Icon(Icons.delete, color: Colors.grey),
-            title: const Text(
-              'Delete This Event',
-              style: const TextStyle(color: Colors.black),
-            ),
-          ))
+      event.creator.id == model.userID
```
Comment (TheHazeEffect) [inline lib/views/pages/events/event_card_widget.dart:108]: recommend using this syntax, you get to cover both buttons without it looking awkward

```dart
if(event.creator.id == model.userID)... [
//Admin Edit Widget here,
//Admin Delete Widget here,
],
```

Comment (AdityaBirangal) [reply lib/views/pages/events/event_card_widget.dart:108]: Thanks 
I learned new thing today :)

**Comment by AdityaBirangal** [comment]
Done 👍🏻
@TheHazeEffect

**Review by TheHazeEffect — APPROVED** [review]
Resolves Issue #832 for the Mobile application

Ready to be merged 

I believe work still needs to be done to check user permission level in the api
to protect against direct requests, Probably best done by checking user's permission level by token.

**Comment by AdityaBirangal** [comment]
Yeah.
API level updates are in progress at PalisadoesFoundation/talawa-api#261

**Pull request merged**